config/configbuilder.go

```diff
@@ -60,8 +63,19 @@ func (builder *servicesConfigBuilder) SetContext(ctx context.Context) *servicesC
 	return builder
 }
 
-func (builder *servicesConfigBuilder) SetHttpTimeout(httpTimeout time.Duration) *servicesConfigBuilder {
-	builder.httpTimeout = httpTimeout
+// Deprecated
+func (builder *servicesConfigBuilder) SetHttpTimeout(dialTimeout time.Duration) *servicesConfigBuilder {
+	log.Warn("SetHttpTimeout is deprecated. Use SetDialTimeout instead.")
+	return builder.SetDialTimeout(dialTimeout)
+}
```

Comment:
Since this is a library, there's probably no need to declare this API as deprecated. Let'e remove it altogether.